clusters/build-clusters/01_cluster/openshift-machine-api/admin_build01-9hdwj-worker-us-east-1a-debug_machine_set.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata:
+  labels:
+    machine.openshift.io/cluster-api-cluster: build01-9hdwj
+  name: build01-9hdwj-worker-us-east-1a-debug
+  namespace: openshift-machine-api
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      machine.openshift.io/cluster-api-cluster: build01-9hdwj
+      machine.openshift.io/cluster-api-machineset: build01-9hdwj-worker-us-east-1a-debug
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        machine.openshift.io/cluster-api-cluster: build01-9hdwj
+        machine.openshift.io/cluster-api-machine-role: worker
+        machine.openshift.io/cluster-api-machine-type: worker
+        machine.openshift.io/cluster-api-machineset: build01-9hdwj-worker-us-east-1a-debug
+    spec:
+      metadata:
+        creationTimestamp: null
+        labels:
+          node-role.kubernetes.io: debug
+          node-role.kubernetes.io/debug: ""
+      taints:
+      - effect: NoSchedule
+        key: node-role.kubernetes.io/debug
```

Comment:
Do you plan to use our CI workload to run on debug workers?
If yes, we should remove the taints.

Reply:
Yes, but we don't want all the worker nodes to be under debug the debug pool. We just want 2 to 3 nodes to be in the debug pool.

Reply:
Fixed, removed the taint.